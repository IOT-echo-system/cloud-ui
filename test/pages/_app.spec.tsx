import {render} from '@testing-library/react'

import type {Router} from 'next/router'
import App from '../../src/pages/_app'

describe('_App page test', () => {
  it('should match with the snapshot', () => {
    const mockRouter = {asPath: '/'} as unknown as Router
    const TestComponent = () => {
      return <div>Mock Component</div>
    }

    const {container} = render(<App router={mockRouter} Component={TestComponent} pageProps={'props'} />)

    expect(container).toMatchSnapshot()
  })
})
