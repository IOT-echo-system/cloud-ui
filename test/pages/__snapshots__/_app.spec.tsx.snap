// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_App page test should match with the snapshot 1`] = `
<div>
  <div
    class="MuiStack-root css-fe8fx7-MuiStack-root"
  >
    <div>
      Mock Component
    </div>
  </div>
</div>
`;
